lib/internal/Magento/Framework/Escaper.php

```diff
@@ -258,8 +259,16 @@ private function escapeAttributeValue($name, $value)
     public function escapeHtmlAttr($string, $escapeSingleQuote = true)
     {
         if ($escapeSingleQuote) {
-            return $this->getEscaper()->escapeHtmlAttr((string) $string);
+            $translateInline = $this->getTranslateInline();
+
+            if ($translateInline->isAllowed()) {
+                return $this->inlineSensitiveEscapeHthmlAttr((string) $string);
+            }
+
+            return $this->getEscaper()
+                ->escapeHtmlAttr((string) $string);
```

Comment:
Use ternary operator to avoid too much early returns

Reply:
Can you, please, confirm that it will look better?
```suggestion
return $translateInline->isAllowed() ?
                $this->inlineSensitiveEscapeHthmlAttr($string) :
                $this->getEscaper()
                    ->escapeHtmlAttr($string);
```

Reply:
@thomas-kl1

Reply:
Yes, like this:

```suggestion
            $translateInline = $this->getTranslateInline();

            return $translateInline->isAllowed()
                ? $this->inlineSensitiveEscapeHthmlAttr((string) $string)
                : $this->getEscaper()->escapeHtmlAttr((string) $string);
```

Reply:
Fixed.